app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,29 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  assignedContacts: null,
+  tags: [],
 };
 
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
+  assignedContacts: yup
+    .object({
+      label: yup.string().required(),
+      value: yup.number().required(),
```

Comment:
Go through the Slack thread once again. I have mentioned about `oneOf`.

Reply:
I've gone through the thread and made changes as below:
```
assignedContact: yup
    .object({
      label: yup
        .string()
        .oneOf(CONTACTS_DROPDOWN_DATA.map(contact => contact.label)),
      value: yup
        .string()
        .oneOf(CONTACTS_DROPDOWN_DATA.map(contact => contact.value)),
    })
    .nullable()
    .required("Assigned contact is required"),
```
where, 
```
CONTACTS_DROPDOWN_DATA = [
  {
    label: "Ronalds Richards",
    value: "ronald-1",
  },
  {
    label: "Jacob Jones",
    value: "jacob-1",
  }
]
```
Is this the only desired way of using `oneOf` validator in this scenario?
OR
Do we have any option like this? (below code is not correct, but hope it conveys the idea- A way of validating the assignedContact as a whole against the list of maps containing the label and value),
```
assignedContact: yup
    .mixed()
    .oneOf(CONTACTS_DROPDOWN_DATA)
```
Can you please provide an idea of what is `mixed()`?
@yedhink

Reply:
>Is this the only desired way of using oneOf validator in this scenario?

@allenjiji The code you've shown is correct. But I prefer to use `object` with `shape` to keep things explicit.

>Can you please provide an idea of what is mixed()?

I have to research on this. Will get back to you when I am free.